src/aioswitcher/api/__init__.py

```diff
@@ -101,15 +101,17 @@ def __init__(
         ip_address: str,
         device_id: str,
         device_key: str,
-        port: int = SWITCHER_TCP_PORT_TYPE1,
         token: Union[str, None] = None,
     ) -> None:
         """Initialize the Switcher TCP connection API."""
         self._device_type = device_type
         self._ip_address = ip_address
         self._device_id = device_id
         self._device_key = device_key
-        self._port = port
+        if self._device_type.protocol_type == 1:
+            self._port = SWITCHER_TCP_PORT_TYPE1
+        else:
+            self._port = SWITCHER_TCP_PORT_TYPE2
```

Comment:
We have a mapping that is now not in use anywhere in the project:
https://github.com/TomerFi/aioswitcher/blob/b72c9de19c7a87ac260dd34f8350caaa0ff019f4/src/aioswitcher/api/__init__.py#L66-L74
Make sure to remove it

As for this, I usually prefer to make it shorter like this:
```suggestion
        self._port = SWITCHER_TCP_PORT_TYPE1
        if device_type.protocol_type == 2:
            self._port = SWITCHER_TCP_PORT_TYPE2
```

Reply:
Oh I didn't notice the mapping..
Should I just use the mapping? or keep what I did and remove the mapping?

Reply:
Remove the mapping

Reply:
It is done.